src/Service.Tests/SqlTests/RestApiTests/Find/FindApiTestBase.cs

```diff
@@ -762,6 +762,23 @@ public async Task FindTest_NoQueryParams_PaginationNextLink()
             );
         }
 
+        /// <summary>
+        /// Validates that a proper nextLink is created for FindMany requests which do not
+        /// restrict results with query parameters. Engine default paging mechanisms are used
+        /// when > 100 records will be present in result set.
```

Comment:
what is the significance of 100 here? because you're setting new values for min max page size, let's specify which constant /configured property is referenced to assert on page size.

Reply:
Fixed the comment. Had copied from earlier test case and missed changing it.